portfolio/src/main/webapp/career.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html>
+   <head>
+      <meta charset="UTF-8">
+      <title>Career</title>
+      <link rel="stylesheet" href="style.css">
+      <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+      <script src="script.js"></script>
+   </head>
+   <body>
+
+      <div id="navigation">
+         <ul>
+            <li><a href="interests.html">Interests</a></li>
+            <li><a href="projects.html">Projects</a></li>
+            <li><a href="career.html">Career</a></li>
+            <li><a href="index.html">About</a></li>
+         </ul>
+      </div>
+
+      <div class="container">
+
+         <div class="title">
+            <h2> Work Experience</h2>
```

Comment:
Nit: extra whitespace in some places, including:

`<h2> Work Experience</h2>` -> `<h2>Work Experience</h2>`

`<h2> Education </h2>` -> `<h2>Education</h2>`

and in other places in this file.